dtfs-central-api/src/helpers/get-keying-sheet-calculation-tfm-facility-values.ts

```diff
@@ -4,34 +4,37 @@ import { TfmFacilitiesRepo } from '../repositories/tfm-facilities-repo';
 import { NotFoundError } from '../errors';
 import { convertTimestampToDate } from './convert-timestamp-to-date';
 import { getEffectiveCoverEndDateAmendment } from './amendments/get-effective-cover-end-date-amendment';
+import { KeyingSheetCalculationTfmFacilityValues } from '../types/tfm/tfm-facility';
 
 /**
- * Gets the latest values for the TFM facility with the supplied facility id
+ * Gets TFM facility values with the supplied facility id for keying sheet calculations
+ * if a report period is supplied, the effective cover end date from the amendment will be applied if the amendment is completed
+ * else will use the snapshot cover end date
```

Comment:
It's not just if the amendment is completed but if the amendment is effective before the end of the report period.
```suggestion
 * if a report period is supplied, the values returned will take into account any amendments that came into effect before the end of the report period passed in.
 * else will return the values as they were at facility creation ignoring all amendments
```

Reply:
changed